sktime/performance_metrics/forecasting/_common.py

```diff
@@ -119,3 +119,45 @@ def _percentage_error(y_true, y_pred, symmetric=False, relative_to="y_true", eps
         denominator = np.maximum(np.abs(y_true), eps)
     percentage_error = np.abs(y_true - y_pred) / denominator
     return percentage_error
+
+
+def _accuracy_ratio(y_true, y_pred, eps=None):
```

Comment:
i think this is not an "accuracy ratio" - it is just a ratio. I would write it as `_fraction(x, y, eps=None)`.